content/about.md

```diff
@@ -17,11 +17,17 @@ Data Together is an ongoing, inclusive conversation, an open source community, a
 
 Data Together was founded and is led by three partner organizations listed below, and comprises members across the world.
 
-<a href="https://envirodatagov.org"><img src="/images/EDGI_Logo.png" width="250" height="250" title="EDGI" alt="edgi"></a>
+<div class="col-md-4 col-sm-12 col-xs-12">
+<a href="https://envirodatagov.org"><img src="/images/EDGI_Logo.png" width="100%" title="EDGI" alt="edgi"></a>
+</div>
 
-<a href="https://qri.io"><img src="/images/blobs_with_large_text_black.png" width="250" height="250" title="Qri" alt="Qri"></a>
+<div class="col-md-4 col-sm-12 col-xs-12">
+<a href="https://qri.io"><img src="/images/blobs_with_large_text_black.png" width="100%" title="Qri" alt="Qri"></a>
+</div>
 
-<a href="https://protocol.ai/"><img src="/images/PL_Logo.png" width="350" height="250" title="EDGI" alt="edgi"></a>
+<div class="col-md-4 col-sm-12 col-xs-12">
+<a href="https://protocol.ai/"><img src="/images/PL_Logo.png" width="100%" title="EDGI" alt="edgi"></a>
+</div>
 
 <a href="/join"> <button type="button">Become a member</button> </a>
```

Comment:
Would it be ok to change this too? "Join" page isn't really offering membership, more like involvement

Reply:
Yea, would be inclined to say "Join the discussion", or "Get involved"